src/frontend/index.html

```
@@ -222,6 +222,68 @@ <h1 class="glitch" data-text="AI TEST AUTOMATION">AI TEST AUTOMATION</h1>
+                <a id="feedback-pass-link" class="feedback-link" href="javascript:void(0)">
+                    Something not right? Let us know →
+                </a>
+            </div>
+
+            <!-- Inline textarea for pass edge case (expanded from link) -->
+            <div id="feedback-pass-detail" class="feedback-pass-detail" style="display: none;">
+                <textarea id="feedback-pass-text" class="feedback-textarea"
+                          placeholder="What could be improved?"
+                          rows="2" maxlength="500"></textarea>
+                <div class="feedback-actions">
+                    <button id="btn-submit-pass-feedback" class="feedback-submit-btn small">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)